src/os/bluestore/BlueStore.cc

```diff
@@ -7375,6 +7397,10 @@ int BlueStore::_mount()
 {
   dout(5) << __func__ << "NCB:: path " << path << dendl;
   _kv_only = false;
+#ifdef NCB_FSCK_DEEP_DEBUG
```

Comment:
to be removed?

Reply:
will be removed, used to debug FSCK with DEEP after mkfs/mount/umount